src/platform/visualc/config.h

```diff
@@ -1,5 +1,8 @@
+/* Project name, lower-case and without spaces */
+#define CANONICAL_PROJECT_NAME "dosbox-staging"
```

Comment:
Perhaps define both `DOSBox Staging` and `dosbox-staging` here, since both are considered (official) project names? IMO `DOSBox Staging` is the usual name whereas `dosbox-staging` is more likely the internal project name.

Reply:
> whereas dosbox-staging is more likely the internal name used by the project (when spaces etc are not desired).

yes, that's why I named it `CANONICAL_PROJECT_NAME`, leaving the public name, `DOSBOX Staging` available for a subsequent define, perhaps just `PROJECT_NAME`.

Reply:
> Perhaps define both `DOSBox Staging`

There are many more places where this is used - and this also makes up larger nouns with the same pattern, like "The DOSBox Staging Team" (as credited in the codec information strings).  In some cases, I'm reluctant to scrub our name because it's less of a branding string (which forks should replace), and actually attributing the work to the team (we wouldn't want the fork's project name to be swapped, in some cases).

This is treading a bit further than the scope of the PR. I'm a bit reluctant to expand this further (I'm going to have backport this to 0.79.1 as well.. 

But this is good stuff for a follow on PR.

Reply:
@kcgen I did not know you decided to define CANONICAL_PROJECT_NAME here as part of this PR. So apparently the PR scope is extended with this change of yours. If you are worried about the PR scope, perhaps you can consider undo this define too, and make a follow PR for such changes.

Reply:
This PR included duplicate uses of the `"dosbox-staging"` string, which you commented would be better if these strings were de-duped.

So I went all the way back to the source - which is Meson's metadata, where `"dosbox-staging"` is actually defines - and I used that.

The PR as-is applies an exact 1:1 de-duping if the `"dosbox-staging"` string (and as such, takes care of the original concern you raised - which was a good concern).

I'm not one to just do a partial measure (taking more effort, and having mixed results), when I can easily do it right with almost the same extra effort.

I would prefer this stand as is, and we carry on.

Reply:
In the earlier Windows installer PR, you suggested to change names such as “dosbox-staging” and “DOSBox Staging” etc in the script, and you appeared to have agreed that the change can be done locally (in the script itself). I really did not know if you would go this far with this PR on the other hand.